resippy/image_objects/earth_overhead/physical_camera/physical_camera_image.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import division
+
+import numpy as np
+from numpy import ndarray
+from resippy.image_objects.earth_overhead.abstract_earth_overhead_image import AbstractEarthOverheadImage
+
+
+class PhysicalCameraImage(AbstractEarthOverheadImage):
```

Comment:
Is this meant to be used as an abstract/base class, or as a concrete image object?

Reply:
A concrete implementation of the AbstractEarthOverheadImage